firewood/src/merkle/node.rs

```diff
@@ -309,10 +314,14 @@ impl Node {
     }
 }
 
+const ENCODED_MAX_LEN: usize = TRIE_HASH_LEN;
+
 #[repr(C)]
 struct Meta {
     root_hash: [u8; TRIE_HASH_LEN],
     attrs: NodeAttributes,
+    encoded_len: [u8; size_of::<u64>()],
```

Comment:
If you look at the individual node-types, whenever I'm encoding a length, I use a type alias so that if you make a change to the size of the thing that encodes the length, you'll get those updates everywhere. 
```rust
type EncodedLen = u64;
```

Then this line becomes
```rust
    encoded_len: EncodedLen,
```